code/modules/cargo/packs.dm

```diff
@@ -797,6 +797,65 @@
 		var/item = pick(contains)
 		new item(C)
 
+/datum/supply_pack/security/armory/western
+	name = "Western Frontier Crate"
+	desc = "Howdy Pardner, this here is the finest collection of frontier gear for the aspiring cowboy, sheriff, or Wild West desperado on this side of the solar system. Unfortunately, we've had to lock this down with Armory access to put the postmaster general at ease."
+	cost = 4000
+	contraband = TRUE
+	max_supply = 3
+	contains = list(/obj/item/ammo_box/c38/box,
+					/obj/item/storage/toolbox/ammo/c38,
+					/obj/item/mob_lasso,
+					/obj/item/clothing/shoes/workboots/mining,
+					/obj/item/clothing/gloves/botanic_leather,
+					/obj/item/clothing/gloves/color/black,
+					/obj/item/clothing/head/cowboy,
+					/obj/item/clothing/head/sombrero,
+					/obj/item/clothing/head/sombrero/green,
+					/obj/item/storage/belt/bandolier/western,
+					/obj/item/gun/ballistic/rifle/leveraction,
+					/obj/item/gun/ballistic/rifle/leveraction)
+	var/wear_outer = list(/obj/item/clothing/suit/apron/overalls,
+					/obj/item/clothing/suit/poncho,
+					/obj/item/clothing/suit/poncho/green,
+					/obj/item/clothing/suit/poncho/red)
+	var/wear_under = list(/obj/item/clothing/under/misc/overalls,
+					/obj/item/clothing/under/misc/overalls,
+					/obj/item/clothing/under/misc/overalls,
+					/obj/item/clothing/under/suit/sl,
+					/obj/item/clothing/under/suit/sl)
+	var/cursed = list(/obj/item/clothing/head/helmet/outlaw,
+					/obj/item/clothing/mask/fakemoustache,
+					/obj/item/clothing/suit/poncho/ponchoshame/outlaw,
+					/obj/item/clothing/under/suit/sl,
+					/obj/item/clothing/shoes/workboots/mining,
+					/obj/item/clothing/gloves/color/black,
+					/obj/item/storage/belt/bandolier/western/filled,
+					/obj/item/gun/ballistic/rifle/leveraction,
+					/obj/item/gun/ballistic/revolver/detective/cowboy,
+					/obj/item/clothing/accessory/holster,
+					/obj/item/paper/crumpled/bloody/cursed_western)
+	crate_name = "western frontier crate"
+
+/datum/supply_pack/security/armory/western/fill(obj/structure/closet/crate/C)
+	if (prob(7) && prob(7) && prob(7)) //0.000343% chance of rolling instead of normal contents //Jackpot Babey!!!
```

Comment:
There once was a game called Pokemon, I'm sure you've heard of it - Pokemon gives a base chance to encounter shiny pokemon at approximately 1 in 8000. This is rare, so rare in fact that most players didn't even know that this was something that could happen until the age of internet knowledge sharing came to be. 

This crate by comparsion is approximately a 1 in 300,000 chance to occur, and players will not encounter nearly as many western crates as a pokemon player would pokemon. Beestation has only recorded about 50,000 rounds since it started counting them more than four years ago. While I get the reference this is such a truly astronomically low chance that it may as well not be here.

Reply:
In hindsight, a near one in a third of a billion chance didn't seems *as large* when looking at just the percent number. Updated to a much more reasonable but still rare 1 in 1000 chance (may still tweak to be more rare, will do some testing when I get home to see the apparent rarity by spawning a fuck load of crates).

Reply:
1 in 1000 is a lot more meaningful of a chance. Players could spend all round trying to get it and fail, but it's not so low it will never reasonably happen at random once in a blue moon.